### PR TITLE
fix(import-notation): parse without duplicates

### LINKED ISSUE #263: Import.notation parse returns duplicates
```js
const { parse: bemParse } = require('@bem/sdk.import-notation');

bemParse('b:button2  m:theme=normal m:theme=clear m:size=m m:type=submit');
/**
 * returns:
 * [ { block: 'button2' },
 * { block: 'button2', mod: { name: 'theme' } }, <---
 * { block: 'button2', mod: { name: 'theme', val: 'normal' } },
 * { block: 'button2', mod: { name: 'theme' } }, <---
 * { block: 'button2', mod: { name: 'theme', val: 'clear' } },
 * { block: 'button2', mod: { name: 'size' } },
 * { block: 'button2', mod: { name: 'size', val: 'm' } },
 * { block: 'button2', mod: { name: 'type' } },
 * { block: 'button2', mod: { name: 'type', val: 'submit' } } ]
 */
```

### PR DESCRIPTION
**Pull request by Yeti-or**
resolves #263

### CODE REVIEW

**Comment by Yeti-or** [comment]
@zxqfox what about dropping `node4`?

**Comment by qfox** [comment]
@Yeti-or Let's drop it after nodejs 10.0 will meet the light.

File: packages/import-notation/index.js [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+    mv : v => v.length ? `=${v.join('|')}` : '',
+    t : t => t ? ` t:${t}` : ''
+};
+
+const btmpl = Object.assign({}, tmpl, {
+    m : m => m ? `${tmpl.mn(m['name'])}${tmpl.mv([m['val']])}` : ''
+});
+
+const BemCellSet = hashSet(cell =>
+    ['block', 'elem', 'mod', 'tech']
```
Comment (qfox) [inline packages/import-notation/index.js:17]: Why you can't use `cell.id`?

File: packages/import-notation/index.js [hunk 2]
```diff
@@ -20,19 +40,19 @@ function parse(importString, ctx) {
     const main = {};
     ctx || (ctx = {});
 
-    return importString.split(' ').reduce((acc, importToken) => {
+    return [...importString.split(' ').reduce((acc, importToken) => {
```
Comment (qfox) [inline packages/import-notation/index.js]: why `[...[]]`?

use `Array.from(...)`

File: packages/import-notation/test/parse.test.js [hunk 3]
```diff
@@ -38,6 +38,15 @@ describe('block', () => {
         ]);
     });
 
+    it('should extract block with same modifiers and several values', () => {
```
Comment (qfox) [inline packages/import-notation/test/parse.test.js]: should not duplicate modifier entity for several separated values ?

File: packages/import-notation/index.js (hunk 1 above)
Comment (Yeti-or) [reply packages/import-notation/index.js:17]: I decided to fix this in different task: https://github.com/bem/bem-sdk/issues/265

**Comment by Yeti-or** [comment]
@zxqfox totally forgot about `Array.from` 👍

**Review by qfox — APPROVED** [review]

**Pull request merged**